linux_cpu_usage.py

```diff
@@ -24,26 +57,33 @@ def get_cpustats_dict(header):
     uptime = 0
     cpustats_dict = {}
     count = 0
+    keys = ('user', 'nice', 'system', 'idle', 'iowait', 'irq', 'softirq')
     for line in stat_data:
         ''' here we have to handle some kind of disk
         first the name than the counters as mentioned
         in the header'''
 
         if line.startswith('cpu '):
+            values = line.split()
+            total_dict = dict(zip(keys, values[1:8]))
             if len(line.strip().split()) == 11:
```

Comment:
Might be interesting to document the `11` here and to hold it in a variable at the top.